src/main/utils/createMetaJson.ts

```
@@ -0,0 +1,23 @@
+import generateSeriesId from './generateSeriesId'
+
+const createMetaJson = (seriesName: string, coverImagePath: string) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)